### DISCUSSION TITLE
Running into an error message in the simulation

### DISCUSSION THREAD

**Discussion by Qijt** [SmileiPIC/Smilei#779] [Q&A] [answered]
Hi, I'm in trouble when I conduct a three-dimensional simulation for a relativistic electron beam passing through a metal tube. When I set a relatively large cell length, the program can run smoothly and output results. However, when I make the cell length smaller and the amount of computation becomes larger, errors occur in the program, and the input file and the error messages are shown in the figure as follows. 
I'm looking for some advice on how to solve this problem.

[input.txt](https://github.com/user-attachments/files/18158721/input.txt)

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 5072 RUNNING AT ec0504
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
[proxy:0:3@ec0516.para.bscc] HYD_pmcd_pmip_control_cmd_cb (proxy/pmip_cb.c:480): assert (!closed) failed
[proxy:0:1@ec0514.para.bscc] HYD_pmcd_pmip_control_cmd_cb (proxy/pmip_cb.c:480): assert (!closed) failed
[proxy:0:2@ec0515.para.bscc] HYD_pmcd_pmip_control_cmd_cb (proxy/pmip_cb.c:480): assert (!closed) failed
[proxy:0:1@ec0514.para.bscc] HYDT_dmxu_poll_wait_for_event (lib/tools/demux/demux_poll.c:76): callback returned error status
[proxy:0:2@ec0515.para.bscc] HYDT_dmxu_poll_wait_for_event (lib/tools/demux/demux_poll.c:76): callback returned error status
[proxy:0:1@ec0514.para.bscc] main (proxy/pmip.c:127): demux engine error waiting for event
[proxy:0:2@ec0515.para.bscc] main (proxy/pmip.c:127): demux engine error waiting for event
[proxy:0:3@ec0516.para.bscc] HYDT_dmxu_poll_wait_for_event (lib/tools/demux/demux_poll.c:76): callback returned error status
[proxy:0:3@ec0516.para.bscc] main (proxy/pmip.c:127): demux engine error waiting for event
srun: error: ec0514: task 1: Exited with exit code 7
srun: error: ec0515: task 2: Exited with exit code 7
srun: error: ec0516: task 3: Exited with exit code 7
[mpiexec@ec0504.para.bscc] HYDT_bscu_wait_for_completion (lib/tools/bootstrap/utils/bscu_wait.c:109): one of the processes terminated badly; aborting
[mpiexec@ec0504.para.bscc] HYDT_bsci_wait_for_completion (lib/tools/bootstrap/src/bsci_wait.c:21): launcher returned error waiting for completion
[mpiexec@ec0504.para.bscc] HYD_pmci_wait_for_completion (mpiexec/pmiserv_pmci.c:197): launcher returned error waiting for completion
[mpiexec@ec0504.para.bscc] main (mpiexec/mpiexec.c:247): process manager error waiting for completion

**Comment by mccoys** [comment, marked as answer]
Exit code 9 generally means your application was killed. Maybe you ran out of time?

**Reply by Qijt** [reply]
Do you mean that the time interval for my output results is too long? Do I need to add more nodes and the number of cores? I'm currently using 4 nodes and 256 cores.

**Reply by mccoys** [reply]
No. I meant that your simulation has probably a limited time decided by the queue manager

**Reply by Qijt** [reply]
Ok, I will try to solve it. Thank you very much.
